pkg/sql/catalog/descpb/privilege.go

```diff
@@ -387,7 +387,7 @@ var SystemAllowedPrivileges = map[ID]privilege.List{
 	// users will be able to modify system tables' schemas at will. CREATE and
 	// DROP privileges are allowed on the above system tables for backwards
 	// compatibility reasons only!
-	keys.JobsTableID:                          privilege.ReadWriteData,
+	keys.JobsTableID:                          privilege.ReadData,
```

Comment:
At one point it was a no-no to change the permissions for existing system tables due to some assertions about existing desc we’d hit after upgrade, but maybe that is what maybefix handles? Not sure.

Also, does the root->node via RBAC trick still work in 20.2?